frontend/integration-tests/views/environment.view.ts

```diff
@@ -6,6 +6,7 @@ const inputs = $$('.pf-c-form-control');
 export const rowsKey = $('[placeholder="name"]');
 export const rowsValue = $('[placeholder="value"]');
 export const deleteBtn = $$('[data-test-id="pairs-list__delete-btn"]').first();
+export const deleteFromBtn = $$('[data-test-id="pairs-list__delete-from-btn"]').first();
```

Comment:
Why not just `$('[data-test-id="pairs-list__delete-from-btn"]')` ?

Reply:
Was just copying the same pattern from above, was not thinking of refactoring logic...

Reply:
Updated as suggested